src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/operations/metric_alert.py

```diff
@@ -2,10 +2,99 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
+from __future__ import print_function
```

Comment:
Ugr, what is this for? Was `print` used in this file?

Reply:
Mmmm...yeah. I can remove it.